qgreenland/config/layers/Environmental management/Mineral and hydrocarbon licenses/mcas_mlsa.py

```diff
@@ -0,0 +1,43 @@
+from qgreenland.config.datasets.mineral_and_hydrocarbon_licenses import (
+    mineral_and_hydrocarbon_licenses as dataset,
+)
+from qgreenland.config.helpers.steps.zipped_vector import zipped_vector
+from qgreenland.models.config.layer import ConfigLayer, ConfigLayerInput
+
+
+layer_params = {
+    'mcas_mlsa_public_all': {
+        'title': 'Public licenses',
+        'description': (
+            """Mining licenses granted by the government of Greenland."""
+        ),
+        'asset_id': 'mcas_mlsa_public_all',
+    },
+    'mcas_mlsa_public_historic': {
+        'title': 'Historic public licenses',
+        'description': (
+            """Historic mining licenses granted by the government of Greenland."""
+        ),
+        'asset_id': 'mcas_mlsa_public_historic',
```

Comment:
Not a big deal, but could just use the key as the  asset_id for these layers!

Reply:
True. I've been avoiding changing any layer IDs as part of migration, even the ones that I think should be changed. Maybe we should also be changing layer IDs while we're in there? What do you think?